code/modules/ghosttrap/trap.dm

```diff
@@ -186,3 +186,14 @@ datum/ghosttrap/drone/transfer_personality(var/mob/candidate, var/mob/living/sil
 	if(!assess_candidate(candidate))
 		return 0
 	drone.transfer_personality(candidate.client)
+
+***********************************
```

Comment:
Needs a `/` here, otherwise it's not commented properly.

So `/***************************`